### PR TITLE
Added basic getting started doku-fu, handcrafted with <3

### PR DESCRIPTION
**Pull request by rubiojr**
one small step for a man...

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+```ruby
+cube = octoc.cubes.find { |c| c.name == 'uber-hubber-cube' }
+```
+
+Creating a new server (instance):
+
+```ruby
+server = octoc.servers.create :name => 'foobar',
+                              :memory => '1024', # FIXME, I assume MB, but need 2 double check
```
Comment (lstoll) [inline docs/getting_started.md]: Yep, it is MB!

**Comment by lstoll** [comment]
:+1: looks good! Might be worth noting the fact that it can drive fusion too at some point.

**Comment by calavera** [comment]
:cool: :+1:

**Comment by rubiojr** [comment]
:shipit:

**Pull request merged**